services/madoc-ts/src/gateway/api.ts

```diff
@@ -687,7 +687,10 @@ export class ApiClient {
   }
 
   async listProjectUpdates(id: string | number, page = 1) {
```

Comment:
This is the authenticated API, we will need a separate one for "listSiteProjectUpdates" that can be requested by anonymous users (the endpoint I sent across)